### DISCUSSION TITLE
Zigbee Network on two floors

### DISCUSSION THREAD

**Discussion by Gushtera** [Koenkk/zigbee2mqtt#18720] [General]
Hello guys,
let me describe my issue and if someone can give me some guidence.
I have 2 Floors:
1st Floor only end devices and 1 Linux with docker container of zigbee2mqtt + mosquitto + Sonoff Zigbee 3.0 Plus dongle
2nd Floor I have 2 Ikea repeaters and a lot of battery powered end devices.

The issue is that on the 2nd floor there is no enough covrage and this is causing issues of pairing and basic functioning of the devices. I have one more Sonoff Zigbee 3.0 Plus dongle and Raspberry Pi. Can someone give me the best setup for that?

**Comment by EvertDekker** [comment]
Add more repeaters or devices that are continuously powered like light bulbs
Use a pi for this is a bit overkill and energy hungry.

**Reply by Gushtera** [reply]
The issue is that the signal is very weak between floors also I have repeaters but for some reason the sensors wants to connect on the week signal to the Coordinator and not to the repeaters. Also for some reason I can't pair any device from 2nd floor where I have 3 repeaters the only way possible is on the 1st floor next to the Coordinator. Everything is very weird.

**Reply by lux73** [reply]
For me not all Repeaters works flawless

Best experience with Hue Bulbs - please note that one (1) faulty Repeater is enough to drop your whole Zigbee Network down 🙁

These are only my personal experience 🧐

**Reply by Gushtera** [reply]
the issue is that I don`t have place for bulbs I`m using strange socket lights also all of the lights are controlled via Fibaro Dimmers because I don`t have N
